cdap-ui/app/directives/timeline/timeline.js

```diff
@@ -507,7 +507,8 @@ angular.module(PKG.name + '.commons')
       appId: '@',
       programType: '@',
       programId: '@',
-      runId: '@'
+      runId: '@',
+      previewId: '@'
```

Comment:
I thought we were going to create a copy of timeline directive to isolate changes to preview. Based on changes to timeline I don't know if we should add those changes to timeline directive being shared between different log viewers.